pkg/scheduler/framework/interface.go

```diff
@@ -587,6 +588,9 @@ type Handle interface {
 
 	// Extenders returns registered scheduler extenders.
 	Extenders() []Extender
+
+	// Parallelize allows for parallelizing N independent pieces of work until done or the context is canceled.
+	Parallelize(ctx context.Context, pieces int, doWorkPiece workqueue.DoWorkPieceFunc)
```

Comment:
Did you consider returning the Parallelizer instead? That way, plugins can store the parallelizer instead of the whole handler if they don't need it.

Just leave it like this if it doesn't simplify anything.

Reply:
Done. to keep consistency with sharedLister etc.